robot_smach_states/src/robot_smach_states/dummy_state.py

```diff
@@ -0,0 +1,48 @@
+import smach
```

Comment:
Perhaps the dummy state should be moved to the util folder to keep the main folder clean

Reply:
File has been moved